streams/src/main/java/org/apache/kafka/streams/processor/internals/StoreChangelogReader.java

```diff
@@ -479,6 +485,26 @@ public void restore(final Map<TaskId, Task> tasks) {
         }
     }
 
+    private void updateStandbyPartitions(final Map<TaskId, Task> tasks,
```

Comment:
I'm wondering if we can make this a more general rule, like:

1) When reader is in Restore_Active state i.e. there are at least one (for sake of simplicity just say we happen to have exactly one) active task which needs restoration, say taskA; then taskA was paused, we should be able to transit to Update_Standby.
2) When reader is in Update_Standby state, and there is one active task say taskA resumed; we should be able to transit to Restore_Active.

I know for now it does not matter since we always pause all tasks with the current APIs, but this is extensible for finer-grained controls in the future.